sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsClient.java

```diff
@@ -454,15 +454,25 @@ public void deleteRelationship(String digitalTwinId, String relationshipId) {
     // Models APIs
     //==================================================================================================================================================
 
+    /**
+     * Creates one or many models.
+     * @param models The list of models to create. Each string corresponds to exactly one model.
+     * @return A List of created models.
+     */
+    @ServiceMethod(returns = ReturnType.COLLECTION)
+    public List<ModelData> createModels(List<String> models) {
+        return digitalTwinsAsyncClient.createModels(models).block();
```

Comment:
nit: we've been following the pattern where each sync API calls it's max arg overload, and the max arg overload calls into the async API; so this could call `createModelsWithResponse(List<String> models, Context context)` sync API with `Context.None`.

Reply:
```
return digitalTwinsAsyncClient.createModelsWithResponse(models, Context.NONE)
            .map(Response::getValue).block();
```

Reply:
?